.github/workflows/issue-triage.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Issue Labeler'
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  issue-triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: github/issue-labeler@v2.5
+        with:
+          repo-token: '${{ secrets.GITHUB_TOKEN }}'
+          configuration-path: .github/issue-labeler.yml
+          not-before: 2022-11-27T07:05:06.752Z
```

Comment:
This is used to prevent the workflow from removing labels that were manually added before this workflow was created.

Reply:
If that's the case, I believe this should be bumped to today (or whenever you're ready, just give me a ping and I'll approve, kodiakhq will merge with my approval).